e2e/libs/backup/backup.py

```
@@ -66,3 +66,13 @@ def cleanup_backup_volumes(self):
 
     def cleanup_system_backups(self):
         return self.backup.cleanup_system_backups()
+
+    def check_snapshot_exists_for_backup(self, volume_name, backup_id,
+                                         exists=True):
+        backup = self.backup.get(backup_id, volume_name)
+        snap_name = backup.snapshotName
+        snapshot_id = self.backup.snapshot.get_snapshot_id(snap_name)
+        snap = self.backup.snapshot.get(volume_name, snapshot_id)
+        snap_exists = False if snap.removed else True
```

Ruff (0.8.2)

Use `not ...` instead of `False if ... else True`

Replace with `not ...`

(SIM211)